CHANGELOG.md

```diff
@@ -1,5 +1,128 @@
 # Node.js ChangeLog
 
+## 2015-12-08, Version 5.2.0 (Stable), @rvagg
+
+### Notable changes
+
+* **build**:
+  - Add support for Intel's VTune JIT profiling when compiled with `--enable-vtune-profiling`. For more information about VTune, see <https://software.intel.com/en-us/node/544211>. (Chunyang Dai) [#3785](https://github.com/nodejs/node/pull/3785).
+  - Properly enable V8 snapshots by default. Due to a configuration error, snapshots have been kept off by default when the intention is for the feature to be enabled. (Fedor Indutny) [#3962](https://github.com/nodejs/node/pull/3962).
+* **crypto**:
+  - Generate a public key when explicitly setting a private key for ECDH (Elliptic Curve Diffie-Hellman) exchange objects (created via `crypto.createECDH(curve_name)`). Also deprecated the `setPublicKey()` method for ECDH objects as this can lead to an inconsistent state. The validity of the keypair is now checked when setting a private key. (Michael Ruddy) [#3511](https://github.com/nodejs/node/pull/3511).
```

Comment:
What you have is good/usable. I'd say it like the following (but, no big deal if you want to use what you have):

Simplifies using ECDH (Elliptic Curve Diffie-Hellman) objects with private keys that are not dynamically generated via `generateKeys()`. The public key is now computed when explicitly setting a private key. Added validity checks to reduce the possibility of computing weak or invalid shared secrets. Also, deprecated the `setPublicKey()` method for ECDH objects as its usage is unnecessary and can lead to inconsistent state. (Michael Ruddy) [#3511](https://github.com/nodejs/node/pull/3511).